specs/operator/addon.spec

```diff
@@ -63,4 +63,15 @@ Importance: Critical
 Pos/Neg: Negative
```

Comment:
Why does this have `05` in ID? 🤔

Reply:
Done

Reply:
nope... we removed IDs 1 to 5 and we shouldn't reuse the IDs. This one should be 09, following ones 10 and 11